### PR TITLE
Add tutorial

### LINKED ISSUE #99: We need a user guide (or series of guides) explaining how to create a dataset schema
Things to consider:
* The guide should address the difference between the *datasets* schema and *dataset* schema
* It should explain how loaders are chosen based on the `format.id` key
* It should have example dataset schemas covering the most common dataset types and dataset directory structures
* It should probably explain how to generate the `sha512sum` value for beginner users

### LINKED ISSUE #247: Add a tutorial

### PR DESCRIPTION
**Pull request by xuhdev**
Also Removed some distracting/outdated info in README and point readers to the tutorial.

Close #99
Close #247

---

## Checklist:

<details open>

<summary></summary>

- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
  minor change.
  - [ ] No
    - [ ] The change in this pull request is minor.
  - [x] Yes
    - Close #99
    - Close #247

For the following questions, only check the boxes that are applicable.

- [x] Ensure that the pull request text above the horizontal line is descriptive.
- [x] Add/update relevant tests.
- [x] Add/update relevant documents.

- Do you have triage permission of this repository?
  - [ ] No
    - You are good.
  - [x] Yes
    - Does this pull request close an issue?
      - [ ] No, this pull request also acts as an issue by itself.
        - [ ] Assign yourself.
        - [ ] Label this pull request.
        - [ ] Put a milestone.
        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
        - [ ] Put this pull request under the appropriate epic on ZenHub.
        - [ ] Put an estimate on ZenHub.
        - [ ] Add dependency relationship with other issues/pull requests on ZenHub.
      - [x] Yes, this pull request addresses an issue and does not act as an issue.
        - [x] Connect this pull request to the underlying issue on ZenHub.
        - [x] DON'T:
          - DON'T assign yourself or anyone else.
          - DON'T label.
          - DON'T put a milestone.
          - DON'T put this pull request under any epic on ZenHub.
          - DON'T put an estimate on ZenHub.
    - [x] When the pull request is ready, request review.
</details>

### CODE REVIEW

**Comment by xuhdev** [comment]
You can view the generated documentation by click on "details" of the "Docs" test. An artifact is available for download.

File: docs/source/tutorial/basic-usage.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Basic Usage
+-----------
+
+To start using PyDAX, first import the package:
+
+.. code-block:: python
+
+   >>> import pydax
+
+This implicitly calls ``pydax.init()`` and initialize PyDAX to be ready to retrieve datasets from the default
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
This implicitly calls ``pydax.init()`` and initializes PyDAX to be ready to retrieve datasets from the default
```

File: docs/source/tutorial/basic-usage.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+repository. (We will learn about switching to a non-default repository in :doc:`create-schema`.) To see available
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
repository (Note: We will learn about switching to a non-default repository in :doc:`create-schema`). To see the available
```

File: docs/source/tutorial/basic-usage.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+datasets and their versions in this repository, run
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
datasets and their versions included in this repository, run
```

File: docs/source/tutorial/basic-usage.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+.. code-block:: python
+
+   >>> pydax.list_all_datasets()
+   {'claim_sentences_search': ('1.0.2',), ..., 'wikitext103': ('1.0.1',)}
+
+To look into some information about a particular dataset, use ``pydax.describe_dataset()``:
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
To look up information about a particular dataset, use the function ``pydax.describe_dataset()`` as shown below:
```

File: docs/source/tutorial/basic-usage.rst [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+   >>> print(pydax.describe_dataset('wikipedia_category_stance'))
+   Description: Wikipedia categories and lists annotated for stance (Pro/Con) towards the concepts
+   Homepage: https://developer.ibm.com/exchanges/data/all/wikipedia-category-stance/
+   Size: 525K
+   Published date: 2019-08-01
+   License: Creative Commons Attribution 3.0 Unported
+   Available subdatasets: full
+
+Use ``pydax.load_dataset()`` to load a dataset. It will first download the specified dataset with the specified version
+(default is the latest version) if it's not already downloaded, and then load it.
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
(default is the latest version) if it's not already downloaded, and then load it in memory.
```

File: docs/source/tutorial/basic-usage.rst [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+   >>> wcs_data = pydax.load_dataset('wikipedia_category_stance')
+
+PyDAX will download the `IBM Debater® Wikipedia Category Stance
+<https://developer.ibm.com/exchanges/data/all/wikipedia-category-stance/>`__ dataset (latest version
+``1.0.2``) if it's not already downloaded, and then load it to the variable ``wcs_data``.
+
+``wcs_data`` is a :class:`dict` that stores the loaded content of the dataset. It has one key ``'full'``, which is the
+identifier of a subdataset, i.e., a subdivision of the whole dataset. ``wikipedia_category_stance`` has only one
+subdataset because it's a small dataset, hence ``wcs_data`` has only one key. Because ``wikipedia_category_stance`` is
+in CSV format, PyDAX automatically loaded it to ``wcs_data['full']`` as a :class:`pandas.DataFrame` object, which is a
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst]:
```suggestion
in CSV format, PyDAX will automatically load the dataset to ``wcs_data['full']`` as a :class:`pandas.DataFrame` object, which is a
```

File: docs/source/tutorial/basic-usage.rst [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+convenient way to manipulate CSV files in Python:
```
Comment (kmh4321) [inline docs/source/tutorial/basic-usage.rst:46]:
```suggestion
convenient way to manipulate CSV files in Python. The code snippet below shows the usage of the function:
```

File: docs/source/tutorial/create-schema.rst [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+   14,164
+   15,172
+   16,174
+   17,175
+
+She wrote a README document: ::
+
+   Dakota's height history. Heights are in centimeters.
+
+   To the extent possible under law, Dakota have dedicated all copyright and related and neighboring rights to this
```
Comment (kmh4321) [inline docs/source/tutorial/create-schema.rst]:
```suggestion
   To the extent possible under law, Dakota has dedicated all copyright and related and neighboring rights to this
```

File: docs/source/tutorial/create-schema.rst [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+
+She then packed the data file and the README to a tarball ``dakota-height-history.tar.gz``:
+
+.. code-block:: console
+
+   $ tar tf dakota-height-history.tar.gz
+   README
+   height.csv
+
+She then uploaded the tarball to https://pydax.readthedocs.io/en/latest/_static/dakota-height-history.tar.gz. To ease
```
Comment (kmh4321) [inline docs/source/tutorial/create-schema.rst]: Can we replace the URL with a hyperlinked text instead? having a full URL in the main page might look a bit cluttery.

**Review by kmh4321 — CHANGES_REQUESTED** [review]
some minor comments

File: docs/source/tutorial/basic-usage.rst (hunk 7 above)
Comment (xuhdev) [reply docs/source/tutorial/basic-usage.rst:46]: This is not really a usage of the function, but just to show the type of the variable created by the previous snippet

**Review by kmh4321 — APPROVED** [review]

**Pull request merged**